openassessment/xblock/test/test_staff_area.py

```diff
@@ -737,6 +758,72 @@ def test_staff_delete_student_state(self, xblock):
         resp = xblock.render_student_info(request)
         self.assertIn("response was not found", resp.body.decode('utf-8').lower())
 
+    @patch('openassessment.xblock.waffle_mixin.WaffleMixin.user_state_upload_data_enabled')
+    @scenario('data/file_upload_missing_scenario.xml', user_id='Bob')
+    @log_capture()
+    def test_staff_area_student_upload_info_from_user_state(self, xblock, logger, waffle_patch):
```

Comment:
The order of these arguments doesn't seem right - shouldn't it be `self, logger, xblock, waffle_patch`?

Reply:
https://github.com/edx/edx-ora2/blob/master/openassessment/xblock/test/base.py#L118-L120
The `xblock` argument is always the first argument after self. I will be updating the code to match the argument order with decorator applied order.

Reply:
Ah, good to know.  Thanks for updating the order of the decorators, I think that makes it less confusing.